src/main/docker/compose-jwt.yaml

```diff
@@ -72,6 +72,9 @@ services:
     environment:
       - CYFACE_AUTH_TYPE=jwt
       - CYFACE_JWK=$CYFACE_JWK
+      # Set to `/` or else the openapi listens by default at `/` but the pre-request `response.header.location` points
+      # to `/api/v4` as `entrypoint.sh` defaults to `CYFACE_API_ENDPOINT=/api/v4`.
+      - CYFACE_API_ENDPOINT=/
```

Comment:
Mir ist noch beim Testen des kompletten Uploads (Pre-Request und Upload-Request) an die Test-Instanz, dass `response.header.location` des Pre-Requests auf `/api/v4` zeigt, weil kein `API_ENDPOINT` definiert ist und somit:
- der collector immer auf `/` lauscht
- aber `entrypoing.sh` per default `api/v4` rein gibt, wodurch der collector weiterhin an `/` startet, da wir die Funktion, das zu ändern heraus genommen haben, aber im `response.header.location` `api/v4` steht

Reply:
I think I'm not fully getting it, but there is a todo in the code to remove the ability for setting the endpoint in the collector. I had the impression that it is still used, since the parameter was still there.

Reply:
We tried to remove it as far as I remember, but had to re-add the `endpoint` parameter as there is no other way for the collector to determine the full `location` to be returned by the pre-request to the client.

Before: `http.endpoint` determined where the collector listens, e.g. `api/v4`.
Currently: The collector always listens at `/`, but in production `nginx` maps the collector to `/api/v4`. This information is injected as `http.endpoint` into the collector, so the collector can return the correct `response.header.location`.

So afaik we don't need (or can) change anything there, beside the fix I already introduced in this PR, setting `CYFACE_API_ENDPOINT=/` for local dev setup, as the collector listens as `/` in the dev setup.